programs-remote-listings/rs-connect-api.php

```diff
@@ -36,12 +36,12 @@ public static function get_base_url()
         $base_domain = 'secure.retreat.guru';
         $http = 'https://';
 
-        if (isset($_SERVER['SERVER_NAME']) && 'programs-remote.dev' == $_SERVER['SERVER_NAME']) {
-            $base_domain = 'programs.dev';
+        if (isset($_SERVER['SERVER_NAME']) && ('rgconnect.test' == $_SERVER['SERVER_NAME'] || 'qa-testing.booking-demo.com' == $_SERVER['SERVER_NAME'])) {
```

Comment:
should use envs eg CIRCLECI / CIRCLECI_URL .... LOCAL / LOCALHOST or something like that